packages/utilities/src/lib/tryParseJSON.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Try parse a stringified JSON string.
+ * @param value The value to parse
+ */
+export function tryParseJSON(value: string): object | string {
+	try {
+		return JSON.parse(value);
+	} catch (err) {
+		return value;
+	}
+}
+
+/**
```

Comment:
@favna do we need to do this or can we just axe it ?

Reply:
Yes, removing functions is a breaking change